README.md

```diff
@@ -1,44 +1,7 @@
-[![Open in Codespaces](https://classroom.github.com/assets/launch-codespace-7f7980b617ed060a017424585567c406b6ee15c891e84e1186181d67ecf80aa0.svg)](https://classroom.github.com/open-in-codespaces?assignment_repo_id=11521210)
-# Sphinx Profile Page
+# Profile Website
 
-This is is a template for a resume/ personal portfolio, built with Sphinx! Based on [Chris Holdgraf's personal site](https://github.com/choldgraf/choldgraf.github.io)
+Welcome to my profile webiste, 
+<!-- enter your target audience after the comma above -->
 
-This template is designed for use in the URI CS TD SSP. 
-
-
-## Tips for Updating the Content of this site
-
-- sidebar variables are defined in `info-.yml` 
-- sidebar formatting for sidebar is in `_templates/hello.html` 
-- variables are used via `html_context`
-- get social links back by removing setting to `navbar_end` in `conf.py` and set values by [example](https://github.com/choldgraf/choldgraf.github.io/blob/main/conf.py#L41)
-
-
-## To work with this repo offline (or in codespaces)
-
-**This requires having python installed then installs a package that helps build the website**
-
-The easiest way to build the website is to use `nox`, which handles all of the environment generation automatically.
-To do so, follow these steps:
-
-1. Install `nox`.
-
-   ```shell
-   pip install -U nox
-   ```
-2. To run a live webserver that will auto-build and reload when you make changes, run:
-
-```shell
-nox -s docs-live
-```
-
-If on Codespaces, use accept the port forwarding and open the forwarded port in a new browser tab to preview your site while you work. 
-
-<!-- 
-Run `nox`
-
-   ```shell
-   nox -s docs
-   ```
-
-this should install a Sphinx environment and build the site, putting the output files in `_build/html`. -->
+Since you're here, you might be: 
+<!-- make a bulleted list of 3 fictional visitors to your site. Include a few detials about them that could impact how you design for them. For each visitor, assign a task or goal they have for visiting your profile website -->
```

Comment:
Who are your 3 fictional visitors? What are some details about them and why are they visiting your site?

Reply:
For your ## Design section, describe how you will get your message across using design. Your description should include details on how you will use visual hierarchy, grouping and similarity.

For your ## Accessibility section, describe how your site will be accessible to users with different impairments? What web accessibility testing will you run to ensure this?